src/Package/Impl/Packages/R/RPackage.cs

```diff
@@ -110,6 +111,9 @@ internal class RPackage : BasePackage<RLanguageService>, IRPackage {
                 return;
             }
 
+            var settings = VsAppShell.Current.ExportProvider.GetExportedValue<IRToolsSettings>() as IRPersistentSettings;
```

Comment:
If we want to export persistent, sure. I wanted all the code to deal with save/load randomly.